src/libclipper/src/rpc_service.cpp

```diff
@@ -137,20 +143,46 @@ void RPCService::manage_service(const string address) {
       poll_timeout = 1;
     }
     zmq_poll(items, 1, poll_timeout);
+
     if (items[0].revents & ZMQ_POLLIN) {
       // TODO: Balance message sending and receiving fairly
       // Note: We only receive one message per event loop iteration
       log_info(LOGGING_TAG_RPC, "Found message to receive");
-
       receive_message(socket, connections, connections_containers_map,
                       zmq_connection_id, redis_connection);
     }
     // Note: We send all queued messages per event loop iteration
     send_messages(socket, connections);
+    auto current_time = std::chrono::system_clock::now();
+    if(std::chrono::duration_cast<std::chrono::milliseconds>(current_time - last_fire_log_time_).count() > LOST_CONTACT_FIRE_LOG_FREQUQNCY_MILLS){
```

Comment:
I don't think this is necessary. We're already logging the loss of container contact on line 175. After logging on line 175, we now correctly remove the container from the receiving history and should not log again. Remove `fire_lost_contact_log()` and its associated timing variable `LOST_CONTACT_FIRE_LOG_FREQUQNCY_MILLS`.

If I'm missing something here, please let me know.